cypress/integration/board.spec.js

```diff
@@ -10,74 +10,129 @@ import boardPage from "../pages/boardPage.json";
 import header from "../pages/header.json";
 
 describe("Board CRUD", () => {
-    it("Visit login page and login", () => {
+    before(() => {
         cy.visit("/login");
         cy.get(loginPage.emailInput).type(data.user.email);
         cy.get(loginPage.passwordInput).type(data.user.password);
-        cy.get(loginPage.loginBtn).click();
-    });
+        cy.get(loginPage.loginBtn)
+            .should("be.visible")
+            .click();
+        cy.url().should("contain", "/my-organizations");
 
-    it("Create organization", () => {
-        cy.get(myOrganizations.addNewOrgSpan).click();
+        cy.get(myOrganizations.addNewOrgSpan)
```

Comment:
Please add some whitespace here for readability, this would look much nicer instead:

```suggestion
        cy.get(myOrganizations.addNewOrgSpan)
            .should("be.visible")
            .click();

        cy.get(orgModal.orgNameInput).type("Cypress org");
        
        cy.get(orgModal.nextBtn)
            .should("be.visible")
            .click();
            
        cy.get(orgModal.nextBtn)
            .should("be.visible")
            .click();
            
        cy.get(modal.okBtn)
            .should("be.visible")
            .click();
```